commit 0beb9542e90ab1c5d1507a1046a326fbcf73861c
Author: Yu Watanabe <watanabe.yu+github@gmail.com>
Date:   Mon Apr 15 20:56:01 2019 +0900

    network: logs link state change

diff --git a/src/network/networkd-link.c b/src/network/networkd-link.c
index 4846b13a8b..49363299e5 100644
--- a/src/network/networkd-link.c
+++ b/src/network/networkd-link.c
@@ -721,6 +721,10 @@ static void link_set_state(Link *link, LinkState state) {
         if (link->state == state)
                 return;
 
+        log_link_debug(link, "State changed: %s -> %s",
+                       link_state_to_string(link->state),
+                       link_state_to_string(state));
+
         link->state = state;
 
         link_send_changed(link, "AdministrativeState", NULL);
