*** systemd-239/meson.build.orig        2018-10-11 14:50:47.269516356 +0900
--- systemd-239/meson.build     2018-10-11 14:50:52.382852318 +0900
***************
*** 764,770 ****

  enable_debug_hashmap = false
  enable_debug_mmap_cache = false
! foreach name : get_option('debug')
          if name == 'hashmap'
                  enable_debug_hashmap = true
          elif name == 'mmap-cache'
--- 764,770 ----

  enable_debug_hashmap = false
  enable_debug_mmap_cache = false
! foreach name : get_option('debug-extra')
          if name == 'hashmap'
                  enable_debug_hashmap = true
          elif name == 'mmap-cache'
