--- work-lunar/mozilla/certdata2pem.py  2021-10-16 18:09:43.000000000 +0200
+++ work/mozilla/certdata2pem.py        2022-08-20 16:41:50.267839089 +0200
@@ -122,7 +122,7 @@
         if not obj['CKA_LABEL'] in trust or not trust[obj['CKA_LABEL']]:
             continue
 
-        cert = x509.load_der_x509_certificate(obj['CKA_VALUE'])
+        cert = x509.load_der_x509_certificate(bytes(obj['CKA_VALUE']))
         if cert.not_valid_after < datetime.datetime.now():
             print('!'*74)
             print('Trusted but expired certificate found: %s' % obj['CKA_LABEL'])
