*** ca-certificates/mozilla/certdata2pem.py     2023-01-14 22:58:27.000000000 +0900
--- ca-certificates.new/mozilla/certdata2pem.py 2023-05-04 19:47:55.809483254 +0900
***************
*** 29,35 ****
  import io

- from cryptography import x509
-
-
  objects = []

--- 29,32 ----
***************
*** 123,132 ****
              continue

-         cert = x509.load_der_x509_certificate(bytes(obj['CKA_VALUE']))
-         if cert.not_valid_after < datetime.datetime.utcnow():
-             print('!'*74)
-             print('Trusted but expired certificate found: %s' % obj['CKA_LABEL'])
-             print('!'*74)
-
          bname = obj['CKA_LABEL'][1:-1].replace('/', '_')\
                                        .replace(' ', '_')\
--- 120,123 ----
