diff -C 4 -r ca-certificates-20190110.orig/sbin/update-ca-certificates ca-certificates-20190110/sbin/update-ca-certificates
*** ca-certificates-20190110.orig/sbin/update-ca-certificates	2019-07-04 13:44:59.913020208 +0900
--- ca-certificates-20190110/sbin/update-ca-certificates	2019-07-04 13:46:45.009898816 +0900
***************
*** 180,192 ****
      if [ "$verbose" = 1 ]; then
        echo "Removed orphan symlink $orphan"
      fi
    done
    if [ "$verbose" = 0 ]
    then
!     openssl rehash . > /dev/null
    else
!     openssl rehash -v .
    fi
  fi
  
  chmod 0644 "$TEMPBUNDLE"
--- 180,200 ----
      if [ "$verbose" = 1 ]; then
        echo "Removed orphan symlink $orphan"
      fi
    done
+   # Added support to handle older openssl and libressl
+   if type c_rehash &> /dev/null; then
+     SSLCMD="c_rehash"
+   elif openssl version | grep -q '^OpenSSL'; then
+     SSLCMD="openssl rehash"
+   else # Assume libressl
+     SSLCMD="openssl certhash"
+   fi
    if [ "$verbose" = 0 ]
    then
!     $SSLCMD . > /dev/null
    else
!     $SSLCMD -v .
    fi
  fi
  
  chmod 0644 "$TEMPBUNDLE"
