--- bash-completion-2.11/bash_completion	2020-07-25 12:49:49.000000000 +0100
+++ bash-completion-2.11-lunar/bash_completion	2020-09-06 13:19:36.452807925 +0100
@@ -959,6 +959,10 @@
         COMPREPLY=($(compgen -W "$(command ls -B \
             /etc/sysconfig/interfaces |
             command sed -ne 's|.*ifcfg-\([^*].*\)$|\1|p')" -- "$cur"))
+    elif [[ -f /etc/lunar.release ]] || grep -q "Lunar Linux" /etc/os-release; then
+        # Lunar Linux
+        COMPREPLY=($( compgen -W "$( command ls -B \
+            /etc/config.d/network)" -- "$cur" ))
     else
         # Assume Red Hat
         COMPREPLY=($(compgen -W "$(printf '%s\n' \
