diff -Naur iproute2-4.1.1/netem/Makefile iproute2-4.1.1-lunar/netem/Makefile
--- iproute2-4.1.1/netem/Makefile	2015-07-06 23:57:34.000000000 +0200
+++ iproute2-4.1.1-lunar/netem/Makefile	2015-07-25 21:59:40.088900998 +0200
@@ -20,9 +20,9 @@
 	$(HOSTCC) $(CCOPTS) -I../include -o $@ $@.c -lm
 
 install: all
-	mkdir -p $(DESTDIR)$(LIBDIR)/tc
+	mkdir -p $(DESTDIR)$(DATADIR)/tc
 	for i in $(DISTDATA); \
-	do install -m 644 $$i $(DESTDIR)$(LIBDIR)/tc; \
+	do install -m 644 $$i $(DESTDIR)$(DATADIR)/tc; \
 	done
 
 clean:
diff -Naur iproute2-4.1.1/tc/tc_util.c iproute2-4.1.1-lunar/tc/tc_util.c
--- iproute2-4.1.1/tc/tc_util.c	2015-07-06 23:57:34.000000000 +0200
+++ iproute2-4.1.1-lunar/tc/tc_util.c	2015-07-25 21:58:47.742232502 +0200
@@ -28,8 +28,8 @@
 #include "tc_util.h"
 #include "tc_common.h"
 
-#ifndef LIBDIR
-#define LIBDIR "/usr/lib"
+#ifndef DATADIR
+#define DATADIR "/usr/share"
 #endif
 
 static struct db_names *cls_names = NULL;
@@ -68,7 +68,7 @@
 
 	lib_dir = getenv("TC_LIB_DIR");
 	if (!lib_dir)
-		lib_dir = LIBDIR "/tc/";
+		lib_dir = DATADIR "/tc/";
 
 	return lib_dir;
 }
