--- ./configure.ac.old	2013-04-16 00:22:24.000000000 +0300
+++ ./configure.ac	2013-04-20 11:12:24.155259923 +0300
@@ -3480,8 +3480,6 @@
 gobject/tests/Makefile
 gthread/Makefile
 gio/Makefile
-gio/gdbus-2.0/codegen/Makefile
-gio/gdbus-2.0/codegen/config.py
 gio/gnetworking.h
 gio/xdgmime/Makefile
 gio/inotify/Makefile
--- ./docs/reference/gio/Makefile.am.orig	2016-10-12 16:48:57.224594892 +0300
+++ ./docs/reference/gio/Makefile.am		2016-10-12 16:52:02.298924980 +0300
@@ -211,9 +211,6 @@
 	glib-compile-resources.1	\
 	gsettings.1			\
 	gresource.1			\
-	gdbus.1				\
-	gdbus-codegen.1			\
-	gio.1				\
 	$(NULL)
 
 XSLTPROC_FLAGS = \

--- ./gio/Makefile.am.orig	2013-10-04 17:37:06.666271325 +0300
+++ ./gio/Makefile.am		2013-10-04 17:38:17.801356614 +0300
@@ -1,6 +1,6 @@
 include $(top_srcdir)/glib.mk
 
-SUBDIRS = gdbus-2.0/codegen
+SUBDIRS = 
 
 if OS_UNIX
 SUBDIRS += xdgmime
--- ./gio/tests/Makefile.am.orig	2014-03-29 01:18:48.392622448 +0200
+++ ./gio/tests/Makefile.am		2014-03-29 01:25:35.679728555 +0200
@@ -494,7 +494,7 @@
 
 BUILT_SOURCES += gdbus-test-codegen-generated.c gdbus-test-codegen-generated.h
 gdbus-test-codegen.o: gdbus-test-codegen-generated.h
-gdbus-test-codegen-generated.h: test-codegen.xml Makefile $(top_builddir)/gio/gdbus-2.0/codegen/gdbus-codegen
+gdbus-test-codegen-generated.h: test-codegen.xml Makefile
 	$(AM_V_GEN) UNINSTALLED_GLIB_SRCDIR=$(top_srcdir) \
 		UNINSTALLED_GLIB_BUILDDIR=$(top_builddir) \
 		$(PYTHON) $(top_builddir)/gio/gdbus-2.0/codegen/gdbus-codegen \
