--- ./configure.ac.old	2013-04-16 00:22:24.000000000 +0300
+++ ./configure.ac	2013-04-20 11:12:24.155259923 +0300
@@ -3481,8 +3481,6 @@
 gobject/tests/Makefile
 gthread/Makefile
 gio/Makefile
-gio/gdbus-2.0/codegen/Makefile
-gio/gdbus-2.0/codegen/config.py
 gio/gnetworking.h
 gio/xdgmime/Makefile
 gio/inotify/Makefile
--- ./gio/Makefile.am.orig	2019-01-21 15:38:50.000000000 +0200
+++ ./gio/Makefile.am		2019-01-23 21:48:53.853104580 +0200
@@ -1,6 +1,6 @@
 include $(top_srcdir)/glib.mk

-SUBDIRS = gdbus-2.0/codegen
+SUBDIRS =

 if OS_UNIX
 SUBDIRS += xdgmime
--- ./gio/tests/Makefile.am.old	2019-01-21 15:38:50.000000000 +0200
+++ ./gio/tests/Makefile.am	2019-01-23 22:01:07.683223160 +0200
@@ -63,7 +63,7 @@
 	socket					\
 	socket-listener				\
 	socket-service				\
-	srvtarget				\
+76;16M	srvtarget				\
 	task					\
 	tls-interaction				\
 	vfs					\
@@ -494,7 +494,7 @@

 BUILT_SOURCES += gdbus-test-codegen-generated.c gdbus-test-codegen-generated.h gdbus-test-codegen-generated-interface-info.c gdbus-test-codegen-generated-interface-info.h
 gdbus-test-codegen.o: gdbus-test-codegen-generated.h gdbus-test-codegen-generated-interface-info.h
-gdbus-test-codegen-generated.h: test-codegen.xml Makefile $(top_builddir)/gio/gdbus-2.0/codegen/gdbus-codegen
+gdbus-test-codegen-generated.h: test-codegen.xml Makefile
 	$(AM_V_GEN) UNINSTALLED_GLIB_SRCDIR=$(top_srcdir) \
 		UNINSTALLED_GLIB_BUILDDIR=$(top_builddir) \
 		$(PYTHON) $(top_builddir)/gio/gdbus-2.0/codegen/gdbus-codegen \
--- ./docs/reference/gio/Makefile.am.orig	2016-10-12 16:48:57.224594892 +0300
+++ ./docs/reference/gio/Makefile.am		2016-10-12 16:52:02.298924980 +0300
@@ -211,9 +211,6 @@
 	glib-compile-resources.1	\
 	gsettings.1			\
 	gresource.1			\
-	gdbus.1				\
-	gdbus-codegen.1			\
-	gio.1				\
 	$(NULL)

 XSLTPROC_FLAGS = \



