--- ./configure.ac.old	2013-04-16 00:22:24.000000000 +0300
+++ ./configure.ac	2013-04-20 11:12:24.155259923 +0300
@@ -3481,8 +3481,6 @@
 gobject/tests/Makefile
 gthread/Makefile
 gio/Makefile
-gio/gdbus-2.0/codegen/Makefile
-gio/gdbus-2.0/codegen/config.py
 gio/gnetworking.h
 gio/xdgmime/Makefile
 gio/inotify/Makefile
--- ./docs/reference/gio/Makefile.am.orig	2016-10-12 16:48:57.224594892 +0300
+++ ./docs/reference/gio/Makefile.am		2016-10-12 16:52:02.298924980 +0300
@@ -211,9 +211,6 @@
 	glib-compile-resources.1	\
 	gsettings.1			\
 	gresource.1			\
-	gdbus.1				\
-	gdbus-codegen.1			\
-	gio.1				\
 	$(NULL)

 XSLTPROC_FLAGS = \


