diff -r -C 2 a/lang/cl/mkerrcodes.awk b/lang/cl/mkerrcodes.awk
*** a/lang/cl/mkerrcodes.awk	2013-03-16 04:24:25.000000000 +0900
--- b/lang/cl/mkerrcodes.awk	2019-11-01 09:47:24.647729585 +0900
***************
*** 123,127 ****
  
  !header {
!   sub (/\#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
--- 123,127 ----
  
  !header {
!   sub (/#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
diff -r -C 2 a/src/Makefile.am b/src/Makefile.am
*** a/src/Makefile.am	2018-12-12 17:14:31.000000000 +0900
--- b/src/Makefile.am	2019-11-01 09:47:24.647729585 +0900
***************
*** 294,298 ****
  errnos-sym.h: Makefile mkstrtable.awk errnos.in
  	$(AWK) -f $(srcdir)/mkstrtable.awk -v textidx=2 -v nogettext=1 \
! 		-v prefix=GPG_ERR_ -v namespace=errnos_ \
  		$(srcdir)/errnos.in >$@
  
--- 294,298 ----
  errnos-sym.h: Makefile mkstrtable.awk errnos.in
  	$(AWK) -f $(srcdir)/mkstrtable.awk -v textidx=2 -v nogettext=1 \
! 		-v prefix=GPG_ERR_ -v pkg_namespace=errnos_ \
  		$(srcdir)/errnos.in >$@
  
diff -r -C 2 a/src/Makefile.in b/src/Makefile.in
*** a/src/Makefile.in	2019-03-19 17:58:04.000000000 +0900
--- b/src/Makefile.in	2019-11-01 09:48:19.787558197 +0900
***************
*** 1616,1620 ****
  errnos-sym.h: Makefile mkstrtable.awk errnos.in
  	$(AWK) -f $(srcdir)/mkstrtable.awk -v textidx=2 -v nogettext=1 \
! 		-v prefix=GPG_ERR_ -v namespace=errnos_ \
  		$(srcdir)/errnos.in >$@
  
--- 1616,1620 ----
  errnos-sym.h: Makefile mkstrtable.awk errnos.in
  	$(AWK) -f $(srcdir)/mkstrtable.awk -v textidx=2 -v nogettext=1 \
! 		-v prefix=GPG_ERR_ -v pkg_namespace=errnos_ \
  		$(srcdir)/errnos.in >$@
  
diff -r -C 2 a/src/mkerrcodes1.awk b/src/mkerrcodes1.awk
*** a/src/mkerrcodes1.awk	2013-03-16 04:24:25.000000000 +0900
--- b/src/mkerrcodes1.awk	2019-11-01 09:47:24.647729585 +0900
***************
*** 82,86 ****
  
  !header {
!   sub (/\#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
--- 82,86 ----
  
  !header {
!   sub (/#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
diff -r -C 2 a/src/mkerrcodes2.awk b/src/mkerrcodes2.awk
*** a/src/mkerrcodes2.awk	2013-03-16 04:24:25.000000000 +0900
--- b/src/mkerrcodes2.awk	2019-11-01 09:47:24.647729585 +0900
***************
*** 92,96 ****
  
  !header {
!   sub (/\#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
--- 92,96 ----
  
  !header {
!   sub (/#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
diff -r -C 2 a/src/mkerrcodes.awk b/src/mkerrcodes.awk
*** a/src/mkerrcodes.awk	2013-03-16 04:24:25.000000000 +0900
--- b/src/mkerrcodes.awk	2019-11-01 09:47:24.647729585 +0900
***************
*** 86,90 ****
  
  !header {
!   sub (/\#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
--- 86,90 ----
  
  !header {
!   sub (/#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
diff -r -C 2 a/src/mkerrnos.awk b/src/mkerrnos.awk
*** a/src/mkerrnos.awk	2013-03-16 04:24:25.000000000 +0900
--- b/src/mkerrnos.awk	2019-11-01 09:47:24.647729585 +0900
***************
*** 84,88 ****
  
  !header {
!   sub (/\#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
--- 84,88 ----
  
  !header {
!   sub (/#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
diff -r -C 2 a/src/mkstrtable.awk b/src/mkstrtable.awk
*** a/src/mkstrtable.awk	2013-03-16 04:24:25.000000000 +0900
--- b/src/mkstrtable.awk	2019-11-01 09:47:24.647729585 +0900
***************
*** 78,82 ****
  # The variable prefix can be used to prepend a string to each message.
  #
! # The variable namespace can be used to prepend a string to each
  # variable and macro name.
  
--- 78,82 ----
  # The variable prefix can be used to prepend a string to each message.
  #
! # The variable pkg_namespace can be used to prepend a string to each
  # variable and macro name.
  
***************
*** 103,107 ****
        print "   optimal code with a minimum of relocations.  */";
        print "";
!       print "static const char " namespace "msgstr[] = ";
        header = 0;
      }
--- 103,107 ----
        print "   optimal code with a minimum of relocations.  */";
        print "";
!       print "static const char " pkg_namespace "msgstr[] = ";
        header = 0;
      }
***************
*** 111,115 ****
  
  !header {
!   sub (/\#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
--- 111,115 ----
  
  !header {
!   sub (/#.+/, "");
    sub (/[ 	]+$/, ""); # Strip trailing space and tab characters.
  
***************
*** 151,155 ****
      print "  gettext_noop (\"" last_msgstr "\");";
    print "";
!   print "static const int " namespace "msgidx[] =";
    print "  {";
    for (i = 0; i < coded_msgs; i++)
--- 151,155 ----
      print "  gettext_noop (\"" last_msgstr "\");";
    print "";
!   print "static const int " pkg_namespace "msgidx[] =";
    print "  {";
    for (i = 0; i < coded_msgs; i++)
***************
*** 159,163 ****
    print "";
    print "static GPG_ERR_INLINE int";
!   print namespace "msgidxof (int code)";
    print "{";
    print "  return (0 ? 0";
--- 159,163 ----
    print "";
    print "static GPG_ERR_INLINE int";
!   print pkg_namespace "msgidxof (int code)";
    print "{";
    print "  return (0 ? 0";
